docs/docker_demo.md

```diff
@@ -890,6 +1046,39 @@ scala> spark.sql("select `_hoodie_commit_time`, symbol, ts, volume, open, close
 +----------------------+---------+----------------------+---------+------------+-----------+--+
 ```
 
+##### Presto queries over Read Optimized View on MOR dataset after compaction
```

Comment:
a step number for this?

Reply:
I thought of having one initially. But since Spark section above is not having a step number. So kept it consistent. Let me add step numbers for both Spark and Presto and update the diff.